Samples/Cordova/ExampleProject/platforms/android/CordovaLib/src/org/apache/cordova/CordovaActivity.java

```diff
@@ -391,6 +391,7 @@ public void onReceivedError(final int errorCode, final String description, final
         if ((errorUrl != null) && (!failingUrl.equals(errorUrl)) && (appView != null)) {
             // Load URL on UI thread
             me.runOnUiThread(new Runnable() {
+                @Override
                 public void run() {
                     me.appView.showWebPage(errorUrl, false, true, null);
```

Comment:
since this code runs async, do we need some hygiene checks here as well like `(appView != null)`

Reply:
These are all auto generated files by cordova